src/main/java/reposense/parser/ArgsParser.java

```diff
@@ -51,8 +51,9 @@ private static ArgumentParser getArgumentParser() {
                 .metavar("PATH")
                 .help("The directory containing the config files.");
 
-        mutexParser.addArgument("-repos")
+        mutexParser.addArgument("-repo", "-repos")
                 .nargs("+")
+                .dest("repos")
```

Comment:
`.dest("repos")` is added so that the value can be retrieved by `repos` key.

Prior to this change, it implicitly use the argument `-repos` as the key `repos` but it does not work well with `multiple` aliases.